packages/protocol/contracts/rrp/authorizers/mock/Api3Token.sol

```diff
@@ -0,0 +1,110 @@
+//SPDX-License-Identifier: MIT
```

Comment:
It is a little bit weird to see the whole implementation under the mocks folder :)

Can we just add a mocked impl for just the functions needed in the tests?

Reply:
yeah I've updated with only the required functions